### PR TITLE
Create this command to open up a new chrome instance with web security disabled

### PR DESCRIPTION
**Pull request by thsnyzkn**
## Description

If you run your frontend project locally and backend restricts to use their api for security reasons, sometimes it is needed to run Chrome without CORS by disabling web security. This command simply helps developers create new instance of  Chrome with specified flags.

## Type of change

- [X] New script command

## Screenshot

chrome-no-cors

## Checklist

- [X] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/browsing/open-chrome-without-cors.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Open Chrome without CORS
```
Comment (dehesa) [inline commands/browsing/open-chrome-without-cors.sh:5]: I would change it to:

```bash
# @raycast.title Open without CORS
```

File: commands/browsing/open-chrome-without-cors.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# @raycast.mode compact
+
+# Optional parameters:
+# @raycast.icon 🤖
+# @raycast.packageName Open Chrome without CORS
```
Comment (dehesa) [inline commands/browsing/open-chrome-without-cors.sh:10]: and here:

```bash
# @raycast.packageName Chrome
```

File: commands/browsing/open-chrome-without-cors.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Open Chrome without CORS
+# @raycast.mode compact
+
+# Optional parameters:
+# @raycast.icon 🤖
```
Comment (dehesa) [inline commands/browsing/open-chrome-without-cors.sh:9]: Optionally (and not necessary for this PR) a nice addition would be to upload a 64x64 Chrome icon under the `/commands/browsing/images` folder and point it out here.

**Review by dehesa — CHANGES_REQUESTED** [review]
Hey @thsnyzkn,

Thanks for the contribution, it looks great. I am just suggesting that you change the title title and package name, because they will appear repeated in the RootSearch.

In Raycast, elements in the RootSearch are displayed as: Icon > Title > Package Name.

**Comment by thsnyzkn** [comment]
Hey @dehesa,
Thank you for reviewing and getting back. I added requested changes. Could you please check them, when you have time?

**Comment by dehesa** [comment]
Looks good! Thank you @thsnyzkn

**Pull request merged**